modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java

```diff
@@ -249,6 +251,28 @@ private void generateModelTests(List<File> files, Map<String, Object> models, St
         }
     }
 
+    /**
```

Comment:
This method generates/copies "static" resources for model build. This is similar to "generateSupportingFiles", but "generateSupportingFiles" and "generateModelTests" are independent "phases".